.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+
+**What type of PR is this?**
+> Uncomment only one and also add the corresponding label in the PR:
+>bug
+>feature
+>cleanup
+>api-change
+>design
+>documentation
+>failing-test
+>unit-test
+>integration-test
+
+**What this PR does / why we need it**:
+
+
+**Does this PR change a user-facing CRD or CLI change?**:
```

Comment:
`Does this PR change a user-facing CRD or CLI?`
`change` at the end is redundant.